lib/core/src/animation/physics.rs

```diff
@@ -0,0 +1,248 @@
+use crate::prelude::*;
+
+use crate::animation::Animator;
+
+use nalgebra::Vector3;
+use nalgebra::zero;
+use crate::traits::HasPosition;
+
+// ======================
+// === DragProperties ===
+// ======================
+
+/// This structure contains air dragging properties.
+#[derive(Default, Clone)]
+pub struct DragProperties {
+    amount : Rc<RefCell<f32>>
+}
```

Comment:
Why `DragProperties` is defined as `RefCell`? Why don't we just keep all properites as plain data. I havent found in code any place you need to modify it from different places at once. The same applies to SpringProperties.